build/images/Dockerfile.policy-recommendation.ubuntu

```diff
@@ -0,0 +1,19 @@
+FROM gcr.io/spark-operator/spark-py:v3.1.1
```

Comment:
todo for a future PR: Consider whether this image can be moved in the antrea dockerhub as we did for the other 3rd party images used by Theia

Reply:
Got it, yes we could tag this image and push into our docker hub too.